chord_drs/routes.py

```diff
@@ -46,14 +48,29 @@ def build_bundle_json(drs_bundle: DrsBundle) -> str:
     return response
 
 
-def build_object_json(drs_object: DrsObject) -> str:
-    response = {
-        "access_methods": {
-            "access_url": {
-                "url": url_for('drs_service.object_download', object_id=drs_object.id, _external=True)
-            },
-            "type": "https"
+def build_object_json(drs_object: DrsObject, inside_container: Optional[bool] = False) -> str:
+    default_access_method = {
+        "access_url": {
+            "url": url_for('drs_service.object_download', object_id=drs_object.id, _external=True)
         },
+        "type": "https"
+    }
+
+    if inside_container:
+        access_methods = [
+            default_access_method,
+            {
+                "access_url": {
+                    "url": drs_object.location
```

Comment:
Wait, does this include `file://`? URL should probably be a proper file URL

Reply:
It doesn't but good point, I'll add it